### PR TITLE
Make content scrollable

### PR DESCRIPTION
**Pull request by izgeorge**
makes some layout changes to make right content scrollable. waiting to hear from design about where we want the right content to align.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+html,
+body,
+#root,
+.App {
+  height: 100%;
+}
+
 html {
   height: 100%;
 }
```
Comment (WesSouza) [inline src/index.css]:
```suggestion
```

You could remove `height` across html, body, and etc, right?

Comment (izgeorge) [reply src/index.css]: removing this causes the height to only be the height of the content:

Screen Shot 2020-03-31 at 10 59 33 AM

File: src/components/layouts/Page/index.js [hunk 2]
```diff
@@ -41,8 +40,8 @@ const renderPageContent = ({
   isHome,
   onBackButtonClick,
 }) => (
-  <div css={isDesktop && styles.rightContainerDesktop}>
-    <div css={styles.content}>
+  <div css={[styles.container, isDesktop && styles.rightContainerDesktop]}>
+    <div css={[styles.content, isDesktop && styles.desktopContent]}>
```
Comment (joshrounsville) [inline src/components/layouts/Page/index.js]: Can we just use are responsive styles to handle this change rather? 
```
content: css({
  height: ['100%', '', 'auto'],
})
```

Comment (izgeorge) [reply src/components/layouts/Page/index.js]: derp sorry 🤦‍♀

File: src/index.css (hunk 1 above)
Comment (WesSouza) [reply src/index.css]: But I thought this rule would cover it:

```html,
 body,
 #root,
 .App {
   height: 100%;
 }```

Comment (izgeorge) [reply src/index.css]: oh yeah you're right. removing!

**Pull request merged**